cpu/efm32/Makefile.include

```diff
@@ -18,16 +20,16 @@ ifeq ($(CPU_ARCH),cortex-m0plus)
 endif
 
 # include CPU family module
-USEMODULE += cpu_$(EFM32_FAMILY)
+USEMODULE += cpu_$(CPU_FAM)
```

Comment:
Could this be moved in a extra Makefile.dep file ?
(same for the pm_layered module below in this file)

Reply:
OK!

Reply:
moved